src/main/java/intefaceAdapters/deleteListing/DeleteListingResponseFormatter.java

```diff
@@ -0,0 +1,31 @@
+package intefaceAdapters.deleteListing;
+
+import useCases.deleteListing.DeleteListingDsRequestModel;
+import useCases.deleteListing.DeleteListingPresenter;
+import useCases.deleteListing.DeleteListingResponseModel;
+
+import java.util.List;
+
+public class DeleteListingResponseFormatter implements DeleteListingPresenter {
+    private final DeleteListingScreenInterface view;
+
+    public DeleteListingResponseFormatter(DeleteListingScreenInterface view) {
+        this.view = view;
+    }
+
+    @Override
+    public void displayUserListings(DeleteListingResponseModel responseModel) {
+        List<DeleteListingDsRequestModel> listings = responseModel.getRequestModel();
+        view.displayUserListings(listings);
+    }
+
+    @Override
+    public void sendFailureMessage(String message) {
```

Comment:
this method is really never called since we don't show a failure message anymore so we can completely remove this method and it's calls from the code

Reply:
oh yeah i totally missed that. Let me remove the method and its calls.